src/app/static/src/app/localStorageManager.ts

```diff
@@ -14,10 +14,11 @@ export const serialiseState = (rootState: RootState): Partial<RootState> => {
         modelOptions: rootState.modelOptions,
         modelOutput: rootState.modelOutput,
         stepper: rootState.stepper,
-        metadata: {...rootState.metadata, plottingMetadataError: null},
```

Comment:
was this line removed by mistake? doesn't seem like it has anything to do with the current ticket...